<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginScreen</class>
 <widget class="QWidget" name="LoginScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>880</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Beanfinder</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>512</width>
     <height>256</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>750</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="beanfinderLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>590</y>
     <width>501</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>BEANFINDER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>750</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QListWidget" name="characterList">
   <property name="geometry">
    <rect>
     <x>845</x>
     <y>20</y>
     <width>331</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="createCharacterButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>700</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Character</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
